### ISSUE TITLE
Podcast stream restarts after pause

### ISSUE THREAD

**Issue by drededison** [clementine-player/Clementine#5106]
when pausing a podcast stream it restarts from the beginning again after a few seconds (cache?).
I've seen open cases (#767) on that one but they are related to radio streams. Podcast streams should start at the point when paused not at the beginning. 

Also as an enhancement it would be great if Clementine to remember last position on stop of a Podcast stream. 

Clemetine version 1.2.3

**Comment by Mariglianov** [comment]
Yeah same here, stoping after few seconds after I paused it.

**Comment by nfshr** [comment]
Yup, same for me. Loops over and over for me, not only ends or skips to the next, but loops within the episode indefinitely.
